external-api/src/external-api/bank-holidays.test.ts

```diff
@@ -0,0 +1,18 @@
+import { isBefore } from 'date-fns';
+
+describe('bank-holidays', () => {
+  /**
+   * If this test fails it means it is time to update the bank holiday backup data.
+   * The backup should be retrieved from https://www.gov.uk/bank-holidays.json.
+   *
+   * After updating the backup, the deadline in the test should be extended
+   * to be 6 months before the date of the final bank holiday in the backup.
+   */
```

Comment:
I think a small CRON job that sends emails and/or maybe a Teams notification would be the ideal solution

But this seems to be the most sensible approach for now

Reply:
FWIW I think we decided on this solution since it was easy to implement, ties it to the code and also is not ignorable. It would be fairly easy to accidentally ignore an email and not update it, especially if the people working on the project/channel of communication changed.

We did consider caching the api responses which would remove the need for manual updates but as is detailed on the ticket it was not deemed worth the development time.

Reply:
Thanks @RLavender98 Concern is that if no-one is working on the project it could be missed

Not a blocker, this is another piece, this solution is great

Reply:
Ah true that is an angle I hadn't considered!